### ISSUE TITLE
One database per user versus Crypto Pouch

### ISSUE THREAD

**Issue by fredguth** [eHealthAfrica/couchdb-best-practices#73]
I am used to the "one database per user" pattern, but I say that one can prevent using one database per user and accomplish per document access permission with crypto pouch.  How would that be? Could you elaborate?

**Comment by jo** [comment]
Yes, this is indeed a possibility. One thing to keep in mind, though: replication performance is a little slow when using filters. So having one giant database for all users would require each user to do filtered replication.

**Comment by fredguth** [comment]
Several applications in the SaaS business model may have one-db-per-tenant with several users (team members)  sharing the same db and controlling access via crypto pouch.  I guess this could avoid filtered replication, but I guess the possible down sides of #76 can still happen.  I am wondering about custom replication, as you suggested.  Is there documentation on this kind of replication?
